mycroft/listener/service.py

```diff
@@ -231,3 +231,8 @@ def run(self):
             self.status.set_error(e)
         self.status.set_stopping()
 
+        
+class SpeechClient(SpeechService):
+    def __init__(self, *args, **kwargs):
+        LOG.warning("SpeechClient has been renamed to SpeechService, it will be removed in 0.1.0)
```

Comment:
SyntaxError: unterminated string literal (detected at line 237)

Reply:
Not sure how I missed that in testing.. PR'd a fix https://github.com/OpenVoiceOS/ovos-core/pull/165